jwala-integration-test/src/test/java/com/cerner/jwala/ui/selenium/operation/DrainTest.java

```diff
@@ -1,14 +0,0 @@
-package com.cerner.jwala.ui.selenium.operation;
```

Comment:
Just to confirm, you removed this because you optimized the code to do the drain else where right ? If yes, nice one.

Reply:
yeah drain is included in respective control operations